Riot/Modules/Room/RoomCoordinator.swift

```diff
@@ -80,6 +80,9 @@ final class RoomCoordinator: NSObject, RoomCoordinatorProtocol {
         if let threadId = parameters.threadId {
             self.roomViewController = ThreadViewController.instantiate(withThreadId: threadId,
                                                                        configuration: parameters.displayConfiguration)
+        } else if parameters.userId != nil {
```

Comment:
Is this really needed? I think the view controller will be created with right parameters even we remove this.

Reply:
Thx for your final review
I agree with you, the session seems provided latter to the view controller (via the `- (void)displayNewDirectChatWithTargetUser:(nonnull MXUser*)directChatTargetUser session:(nonnull MXSession*)session` method)
I will check this point with @yostyle to have a third point of view before cleaning the code here

Reply:
You right session parameter is not needed. I removed it